packages/interaction-kit/src/interactions/index.ts

```diff
@@ -96,14 +82,14 @@ async function handleApplicationCommandInteraction(
 		);
 
 		console.log(`Handling ${interaction.name}`);
-		return command.handler(interaction, application);
+		command.handler(interaction, application);
+	} else {
+		throw new Error(
+			// @ts-expect-error TS doesn't think this will happen, but theoretically it can
```

Comment:
```suggestion
			// @ts-expect-error With the current given types, this cannot happen, but Discord may add types at any time
```

This one seems more accurate to me